### DISCUSSION TITLE
Github account flagged as spam - please help

### DISCUSSION THREAD

**Discussion by sailepradh** [community/community#121218] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi,

Please can someone kindly help with my github account (ryanjameskennedy) which was incorrectly flagged as spam? I had to ask a friend to make this post as my account isn't authorized to post on the Github community platform. I use my account heavily for development in my hospital and my teammates cannot see my PRs, issues or reviews, which is incredibly frustrating as we planned to release an open source product this week that many hospitals need access to.
Furthermore, I have submitted a reinstatement request numbered #2741266. Thank you for your help in advance, and please don’t hesitate to let me know if you have any questions.

Kind regards,
Ryan Kennedy

**Comment by q1isnotq1** [comment]
I have same issue.

**Comment by KodenOps** [comment]
I have the same issue. I can't even see my profile from another github account

**Comment by nirmalyax** [comment, marked as answer]
I can advice you to communicate with the Github Support.
My friend had almost the same issue and recovered his account with the help of github support.

**Reply by KodenOps** [reply]
I've logged a ticket for like three weeks now but it hasn't been treated

**Reply by nirmalyax** [reply]
The only option is you need to wait. 
There are millions of ticket in their bucket and it takes time process.
Until your account gets recovered you can backup your codes create a new account and move on with that.

**Reply by ghost** [reply]
@nirmalyax what if all the tickets related to reinstatement request get closed?

P.S my issue is here: https://github.com/orgs/community/discussions/121640

![image](https://github.com/community/community/assets/167653560/571eeb74-a5d0-4fcb-944c-6755957a0c90)

**Reply by nirmalyax** [reply]
I have never encountered with this type of issue.
Looks like something fissy is going on with your account. 
Whats the reason for your account to get locked?

**Reply by ghost** [reply]
@nirmalyax it is not locked, I can sign in, but there is a label `spammy` next to my username.

I have read tons of discussions about how github support can help me, but as you can see... I wish there was at least a note on what I am doing wrong

**Reply by nirmalyax** [reply]
But for me when I try to visit, ERROR 404 not found.
As you have created multiple ticktes might be that is the reason for getting the label `spammy`. 
I am not getting the resaon for others not able to visit your acc.

**Reply by ghost** [reply]
@nirmalyax My original account is @turytsia. Can you visit it?

https://github.com/turytsia

**Reply by nirmalyax** [reply]
@aturytsia or might be you have sent the same message in the dicussion repeatedly and they reported you.

**Reply by ghost** [reply]
@nirmalyax I've never used github discussions until now on [turytsia](https://github.com/turytsia) account

**Reply by nirmalyax** [reply]
No bro its popping up the error message.
<img width="1440" alt="Screenshot 2024-05-01 at 12 34 15 PM" src="https://github.com/community/community/assets/35608135/14621bd7-62fc-41ea-b4ab-94a859b5cb76">

**Reply by nirmalyax** [reply]
Brother @aturytsia dont worry I have some connections, I will try to figure it out.
and for suggestion if you are having some imp repos please clone it into your local.
Don't loose hope :)

**Reply by ghost** [reply]
@nirmalyax thanks for the words of hope

**Comment by yogeshxd** [comment]
same issue here. github should really do something about this.

**Comment by ebndev** [comment]
Hi @sailepradh,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion and any duplicate discussions.

We appreciate your understanding!
